File: CarShop_Mobile/app/src/main/java/com/example/carshop_mobile/main/Model/PickUp.kt
package com.example.carshop_mobile.main.Model

import com.google.gson.annotations.SerializedName

class PickUp() {
    @SerializedName("idPickUp")
    var idPickUp:Int? = null
    @SerializedName("address")
    var address : String = ""
    @SerializedName("longitude")
    var longitude:Double? = null
    @SerializedName("latitude")
    var latitude:Double? = null
    @SerializedName("done")
    var done:Boolean = false
    @SerializedName("user")
    var user:UserMobile? = null

    constructor(address:String,longitude:Double,latitude:Double,done:Boolean,user:UserMobile) : this(){
        this.address = address
        this.longitude = longitude
        this.latitude = latitude
        this.done = done
        this.user = user
    }
}

File: CarShop_REST_API/CarShop_REST_API/Model/PickUp.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace CarShop_REST_API.Model
{
    public class PickUp
    {
        [Key]
        public int IDPickup { get; set; }
        public string Address { get; set; }
        public double Longitude { get; set; }
        public double Latitude { get; set; }
        public bool Done { get; set; }
        public UserMobile User { get; set; }

    }
}
